iterative/utils/analytics.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 var (
-	Version string = "v0.0.0"
+	Version string = "0.0.0"
```

Comment:
We need to change also the test 

```
func TestVersion(t *testing.T) {
	assert.Equal(t, strings.HasPrefix(Version, "v"), true)
}
```

Reply:
https://github.com/iterative/terraform-provider-iterative/pull/537/commits/f0adf73e218436e734a0b71a4091e8cee76412aa